osu.Game/Screens/Menu/LogoVisualisation.cs

```diff
@@ -1,123 +1,31 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-using osuTK;
 using osuTK.Graphics;
-using osu.Framework.Graphics;
-using osu.Framework.Graphics.Batches;
-using osu.Framework.Graphics.Colour;
-using osu.Framework.Graphics.OpenGL.Vertices;
-using osu.Framework.Graphics.Primitives;
-using osu.Framework.Graphics.Shaders;
-using osu.Framework.Graphics.Textures;
-using osu.Game.Beatmaps;
-using osu.Game.Graphics;
 using osu.Game.Skinning;
 using osu.Game.Online.API;
 using osu.Game.Users;
-using System;
 using osu.Framework.Allocation;
 using osu.Framework.Bindables;
 using osu.Framework.Extensions.Color4Extensions;
-using osu.Framework.Utils;
 
 namespace osu.Game.Screens.Menu
 {
-    public class LogoVisualisation : Drawable, IHasAccentColour
+    public class LogoVisualisation : BasicLogoVisualisation
```

Comment:
Yeah the naming isn't too fortunate. I'd probably rename:

* `BasicLogoVisualisation` -> `LogoVisualisation`
* `LogoVisualisation` -> `SkinnableLogoVisualisation`

Reply:
Although now I think about it it might be a breaking change for rulesets... At least two or three custom rulesets use this class on their playfield.

Reply:
The change you've proposed doesn't sound breaking because I don't think the skinned one should have been exposed in the first place (and the name still matches).

Also how about calling the skinned one `MenuLogoVisualisation` and making it internal while we're here?

Also please fully xmldoc the base class if possible.

Reply:
Alright! Will do.